lib/server/index.js

```diff
@@ -52,6 +52,25 @@ class GameServer {
     process.stdout.write(`Connection from ${socket.conn.remoteAddress}\n`);
   }
 
+  listGames() {
+    return {
+      games: Object.keys(this.games).map((id) => {
+        const game = this.games[id];
+
+        return {
+          id,
+          mode: game.mode,
+          playerCount: game.playerCount,
+          maximumPlayerCount: game.maximumPlayerCount,
+        };
+      }),
+    }
+  }
+
+  viewGame(id) {
```

Comment:
Named after the API endpoint. The functionality is simple enough so I'll move it there verbatim.